panic: runtime error: invalid memory address or nil pointer dereference if go_package is not specified

**Describe the bug**
We are not using `go_package` option for our definitions. 

When trying to generate classes from such kind of proto this error appears:
```
gigi:scripts user$ protoc -I . --twirp_php_out=generated --php_out=generated ./service.proto
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x88 pc=0x1384363]

goroutine 1 [running]:
github.com/twirphp/twirp/protoc-gen-twirp_php/internal/php.Namespace(0xc0001240f0, 0xc0000d9440, 0x1)
        /root/project/protoc-gen-twirp_php/internal/php/func.go:68 +0x33
github.com/twirphp/twirp/protoc-gen-twirp_php/internal/gen.(*generator).Generate(0xc0000c5ec0, 0xc0000c9810, 0xc0000a2360, 0x0, 0x0)
        /root/project/protoc-gen-twirp_php/internal/gen/generator.go:64 +0x364
main.Main(0x151fec0, 0xc00000e010, 0x151fee0, 0xc00009c000, 0x148ef9b, 0xb, 0x17d6cb4, 0x22, 0xc0000c5890, 0x0, ...)
        /root/project/protoc-gen-twirp_php/main.go:63 +0x1eb
main.main()
        /root/project/protoc-gen-twirp_php/main.go:35 +0x17a
--twirp_php_out: protoc-gen-twirp_php: Plugin failed with status code 2.

```

**To Reproduce**
1. Remove go_package from .proto definition described here https://twirphp.readthedocs.io/en/latest/getting-started/usage.html.
2. run `protoc -I . --twirp_php_out=generated --php_out=generated ./service.proto`


**Expected behavior**
just generate classes for php

**Environment**
- Version: 0xc0000caaa4
- Go version: go1.12.4 darwin/amd64
- PHP version: PHP 7.3.5

--- a/protoc-gen-twirp_php/internal/php/func.go
+++ b/protoc-gen-twirp_php/internal/php/func.go
@@ -62,11 +62,11 @@ func ClassNamePrefix(className string, file *descriptor.FileDescriptorProto) str
 // 1. Explicitly set namespace using the "php_namespace" option
 // 2. Package name with dots replaced with backslashes and segments converted to title
 func Namespace(file *descriptor.FileDescriptorProto) string {
-	options := file.GetOptions()
-
-	// When there is a namespace option defined we use it
-	if options.PhpNamespace != nil {
-		return options.GetPhpNamespace()
+	if options := file.GetOptions(); options != nil {
+		// When there is a namespace option defined we use it
+		if options.PhpNamespace != nil {
+			return options.GetPhpNamespace()
+		}
 	}
 
 	return Name(file.GetPackage())

